tach.toml

```
@@ -5,7 +5,7 @@ exclude = [
      "docs/**",
      "**/tests/**",
      "tach.egg-info/**",
-     "venv/**",
+     ".venv/**",
```

```suggestion
     "venv/**",
```

Hidden files and folders are automatically excluded